core/rawdb/schema.go

```diff
@@ -89,6 +89,7 @@ var (
 	SnapshotAccountPrefix = []byte("a") // SnapshotAccountPrefix + account hash -> account trie value
 	SnapshotStoragePrefix = []byte("o") // SnapshotStoragePrefix + account hash + storage hash -> storage trie value
 	CodePrefix            = []byte("c") // CodePrefix + code hash -> account code
+	blockResultPrefix     = []byte("e") // blockResultPrefix + hash -> blockResult
```

Comment:
use `T` for trace? or maybe `R` for Result?

Reply:
Use `T` maybe mutch better. already  changed!